index.html

```diff
@@ -1105,6 +1105,57 @@
   This section has not been written.
 </aside>
 
+
+## Principles for Consent, Withdrawal of Consent, Opt-Outs, and Objections
+
+<div class="practice">
+<p>
+  <span class="practicelab" id="principle-consent-user-preference">
+    When any [=actor=] obtains [=consent=] for processing from a [=person=], the
+    actor should design the consent request so as to learn the person's true
+    intent to consent or not, and not to maximize the processing consented to.
+  </span>
+</p>
+</div>
+
+Attempts to obtain consent to [=processing=] that is not in accordance with the person's
+true preferences result in imposing unwanted [=privacy labour=] on the person, and may
+result in people erroneously giving consent that does not match their true preferences.
+
+<div class="practice">
+<p>
+  <span class="practicelab" id="principle-minimize-consent-requests">
+    An [=actor=] should avoid interrupting a [=person=]'s use of a site for
+	consent requests when an alternative is available.
+  </span>
+</p>
+</div>
+
+Examples of alternatives to consent requests include:
+
+ * Considering the information sharing norms in the site's audience
+   and category, and requesting only consent that is appropriate
+   to the purpose of the site. (For example, a photo sharing site's
+   users might expect to be prompted for consent to share their
+   uploaded work.)  Sites should consider conducting user research on
+   people's expectations for how data is processed.
+
+ * Relying on a [=global opt-out=] signal from the [=user agent=].
+
+ * Delaying a prompt for consent until a [=user=] is likely to be sufficiently familiar
+   with the site to give an informed response.
+
+<div class="practice">
+<p>
+  <span class="practicelab" id="principle-consent-withdraw">
+    It should be as easy, or easier, for a user to check what consent they have given
+	for a particular [=context=], or to withdraw consent, or to opt out or object,
+	as to give consent.
```

Comment:
"Or easier" is nice, but maybe not necessary, and shorter is better. :)

```suggestion
    It should be as easy for a user to check what consent they have given,
	to withdraw consent, or to opt out or object,
	as to give consent.
```